Exercise3/main.tf

```diff
@@ -0,0 +1,23 @@
+module "vpc" {
+  source = "./modules/vpc"
+  total_subnets = 3
+  client_name = "rmodule"
+  env = "prod"
+  vpc_cidr = "17.0.0.0/16"
```

Comment:
Try to use the values from the variables files rather than hardcode them directly in the module